package ca.aversa.insessionservice.exception

class UserInvitationBoException(message: String? = null, cause: Throwable? = null): RuntimeException(message, cause) {
}